Essentials/src/com/earth2me/essentials/commands/Commandnear.java

```diff
@@ -7,8 +7,12 @@
 import org.bukkit.Server;
 import org.bukkit.World;
 
+import javafx.util.Pair;
```

Comment:
Using JavaFX's `Pair` class requires that JavaFX is actually present on the server, which as far as I can tell [isn't guaranteed if the server is running OpenJDK](https://stackoverflow.com/questions/35974003/javafx-comes-with-jdk-8#answer-35998804). I'd rather we used Apache Commons' `Pair` - Commons included in server jars and we already use it elsewhere.